.github/workflows/renew_certificate.yaml

```diff
@@ -0,0 +1,47 @@
+name: Renew the CA-verified Certificate
```

Comment:
The Waltz Public Instance is on a regular EC2 instance, not on EKS (like the Legend Public instance). Which means that this action will not work in this case, we don't have the same means of connecting to it. Instead, the connection should be made similarly to how you've implemented the ``refresh_deployment`` github action.

Reply:
Updated accordingly.